bin/php/Dockerfile

```diff
@@ -72,6 +73,13 @@ RUN ./configure
 RUN make
 RUN make install
 
+# Compile the bcmath extension because it cannot be installed with pecl
+WORKDIR /php-src-$PHP_VERSION/ext/bcmath
+RUN phpize
+RUN ./configure
+RUN make
+RUN make install
+
```

Comment:
Thank you for the PR! Do you know why/if it is necessary to compile the extension given there is `--enable-bcmatch` above?

In the documentation it says to use `--enable-bcmatch` and that's it (https://secure.php.net/manual/en/bc.installation.php).